exercises/Dplyr-link-to-databases-R.md

```diff
@@ -5,34 +5,38 @@ title: Link to Databases
 language: R
 ---
 
-Let's access an SQL database directly from R. Install the `RSQLite` package (and 
-the `dplyr` package if you haven't already).
+Let's access an SQL database directly from R using `dplyr`.
 
 Either use an existing copy of the `portal_mammals.sqlite` database or [download
 a new copy](https://ndownloader.figshare.com/files/2292171). You
-should now be able to link to the surveys table in the database using:
+should be able to link to the `surveys` table in the database using:
 
 ```
 portaldb <- src_sqlite("portal_mammals.sqlite")
 surveys <- tbl(portaldb, "surveys")
 ```
 
-`surveys` will be a `tbl_df`, which means that we won't need to worry about it
-printing out huge numbers of rows in the answers below.
+`surveys` will be a `tbl_sqlite`, which means that the table remains external to 
+the R environment. *Also, we won't need to worry about it printing out huge 
+numbers of rows when we look at it.*
 
-Using this table in the database:
-
-1. Use the `nrow()` function to determine how many records are represented in this 
-   dataset.
-2. Select the year, month, day, and species_id columns in that order
-3. Create a new data frame with the year, species_id, and weight in kilograms 
-   of each individual, with no null weights.
-4. Use the `distinct()` function to print the species_id for each
-   species in the dataset that have been weighed.
-5. Calculate the average size of a *Neotoma Albigula* (`NL`) in this dataset.
-6. Create a new data frame with the number of individuals counted in each year
-   of the study. If you don't know how to count things using `dplyr` I'd
-   recommend checking out the
+1. Use the `nrow()` function to determine how many records are represented in
+   `surveys`.
+2. Select (*and print*) the `year`, `month`, `day`, and `species_id` columns in
+   that order.
+3. Create a new data frame with the `year`, `species_id`, and weight in
+   kilograms of each individual, with no null weights.
+4. Use the `distinct()` function to print the `species_id` for each
+   species in the dataset that have been weighed. Name this table `species_ids`.
+5. Determine how many rows are in `species_ids`. *You will have to take this
+   `<derived table>` and make it a local data frame (`tbl_df`) using
+   `collect()`.*
+6. Calculate the average size of a *Neotoma Albigula* (`NL`) in `surveys` stored
+   in a local data frame.
+7. Create a local data frame with the number of individuals counted in each year
+   of the study. *If you don't know how to count things using `dplyr`, I'd
+   recommend checking back at the
```

Comment:
I've generally started trying to avoid the first person in the materials. Maybe "Take another look at the..."